ledger-tool/src/main.rs

```diff
@@ -2504,13 +2506,29 @@ fn main() {
                 })
                 .map(|(pubkey, account, slot)| (pubkey, (account, slot)))
                 .collect();
+            measure.stop();
+            info!("{}", measure);
 
             let print_account_contents = !arg_matches.is_present("no_account_contents");
-            if print_account_contents {
-                println!("Getting accounts contents...");
-                let print_account_data = !arg_matches.is_present("no_account_data");
-                for (pubkey, (account, slot)) in accounts.into_iter() {
-                    let data_len = account.data().len();
+            let print_account_data = !arg_matches.is_present("no_account_data");
+            let rent_collector = bank.rent_collector();
+            let mut total_accounts_stats = bank::TotalAccountsStats::default();
```

Comment:
Why not just import `TotalAccountsStats` at the top of this file?

Reply:
No reason. Fixed.